cpp/tensorrt_llm/nanobind/batch_manager/kvCacheManagerV2Utils.cpp

```
@@ -34,6 +36,39 @@ using SizeType32 = tensorrt_llm::runtime::SizeType32;
 {
 
+namespace
+{
+
+void checkTensorReadableByGpu(at::Tensor const& tensor, at::Device const& outputDevice, char const* name)
+{
+    void const* const inputPointer = tensor.const_data_ptr();
+    cudaPointerAttributes attributes{};
+    cudaError_t const status = cudaPointerGetAttributes(&attributes, inputPointer);
+    TLLM_CHECK_WITH_INFO(status == cudaSuccess, "%s pointer %p is not readable by output device %s: %s", name,
+        inputPointer, outputDevice.str().c_str(), cudaGetErrorString(status));
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If NB_MAKE_OPAQUE is a macro then please configure it.

(unknownMacro)